translator/trace/otlp/otlp_to_oc_test.go

```diff
@@ -0,0 +1,172 @@
+package otlp
+
+import (
+	"testing"
+
+	occommon "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	ocresource "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	octrace "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	commonpb "github.com/open-telemetry/opentelemetry-proto/gen/go/common/v1"
+	resourcepb "github.com/open-telemetry/opentelemetry-proto/gen/go/resource/v1"
+	tracepb "github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
+)
+
+func TestResourceSpansToTraceData(t *testing.T) {
```

Comment:
Only added tests for a public method. We probably don't need exhaustive testing since this is a short-living code.